src/styles/partials/components/_icon-button.scss

```diff
@@ -29,3 +29,65 @@
 		background-color: $vibrant-blue;
 	}
 }
+.dg_icon-buttonDefault {
+	color: $black;
+	display: block;
+	font-family: $default-heading-font;
+	font-weight: $font-bold;
+	text-align: center;
+	text-decoration: none;
+	text-transform: uppercase;
+	border: 2px solid $black;
+	padding: 50px 30px 50px 30px;
+	transition: all 0.3s ease 0s;
+	i,
+	svg {
+		font-size: 60px;
+		color: $gold;
+		display: block;
+		margin: auto;
+		padding: 0 10px 20px;
+	}
+
+	&:hover,
+	&:hover i,
+	&:hover svg {
+		color: $white;
+	}
+
+	&:hover {
+		background-color: $vibrant-blue;
+	}
+}
+.dg_icon-buttonAlternative {
+	color: $white;
```

Comment:
a lot of this is repeated, you should be able to do

```css
.dg_icon-button {
  // shared styles go here
}

.dg_icon-button--default {
  // styles that only apply to default
}

.dg_icon-button--alternative {
  // styles that only apply to default
}
```

Reply:
your component would then have, dg_icon-button and dg_icon-button--modifier clasess